pkg/codegen/docs/templates/examples.tmpl

```diff
@@ -0,0 +1,25 @@
+{{ define "examples" }}
+## Example Usage
+
+{{ range . }}
+### {{ .Heading }}
+
+{{ htmlSafe "{{< langchoose nojavascript >}}" }}
```

Comment:
For a subsequent change, are we going to want to show the JavaScript tab (and just use the TypeScript content for it)?

Reply:
I am actually not sure of it at the moment. It is possible that, for the examples section we may (or must?) show the JS tab. Once the examples are available in the Pulumi schema, I will know for certain.